src/steps/group.ts

```diff
@@ -0,0 +1,33 @@
+import type {
+  Serializable,
+  ToJsonSerializationOptions,
+} from '../serialization';
+import { toJsonSerializationDefaultOptions } from '../serialization';
+import { Pipeline, PotentialStep } from '../index';
+import { LabeledStep } from '../base';
+
+export class GroupStep extends LabeledStep implements Serializable {
```

Comment:
Hey @alefoll 👋 wondering what's blocking this PR from getting merged besides a review? I'm looking to use the group step in one of my pipeline changes.

Reply:
Hey, Joscha is on leave until next year. I think the CI team will make a Canva fork soon

Reply:
@uroojayub-canva, I'm back from holiday and I moved it to : https://github.com/Canva/buildkite-graph/pull/1

Reply:
Yay thanks, i'm following along